salt/beacons/log.py

```diff
@@ -91,21 +104,24 @@ def beacon(config):
         fp_.seek(loc)
 
         txt = fp_.read()
+        log.info('txt {}'.format(txt))
 
         d = {}
-        for tag in config:
-            if 'regex' not in config[tag]:
+        for tag in _config.get('tags', {}):
+            if 'regex' not in _config['tags'][tag]:
                 continue
-            if len(config[tag]['regex']) < 1:
+            if len(_config['tags'][tag]['regex']) < 1:
                 continue
             try:
-                d[tag] = re.compile(r'{0}'.format(config[tag]['regex']))
-            except Exception:
+                d[tag] = re.compile(r'{0}'.format(_config['tags'][tag]['regex']))
+            except Exception as e:
+                log.debug('Exception {}'.format(e))
                 event = SKEL.copy()
                 event['tag'] = tag
                 event['error'] = 'bad regex'
                 ret.append(event)
 
+        log.debug('d {}'.format(d))
```

Comment:
The log calls you added aren't self explanatory when you get them in the middle of a bunch of log records. Perhaps expand them to have some more meaning?